src/snowflake/snowpark/files.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2012-2022 Snowflake Computing Inc. All rights reserved.
+#
+"""SnowflakeFile for UDFs in Snowpark."""
+from __future__ import annotations
+
+import array
+from typing import Iterable
+
+from snowflake.snowpark._internal.utils import private_preview
+
+_DEFER_IMPLEMENTATION_ERR_MSG = (
+    "The file operations is implemented in Snowflake servers"
```

Comment:
This text may be vague and misleading.
1) "file operations" is broad. The SDK already has `session.file.put/get/put_stream/get_stream`.  They're also file operations.
2) Users may be wondering what part of "Snowflake servers". How about explicitly saying "SnowflakeFile currently only in UDF and Stored Procedures. It doesn't work locally yet. "

@sfc-gh-mabrennan, what do you think?

Reply:
Would something like this make sense?

"SnowflakeFile only works in UDFs and stored procedures. It doesn't work locally."

Reply:
thank you, that sounds better, I will say "SnowflakeFile currently only works in UDF and Stored Procedures. It doesn't work locally yet." since we are planning to support local file access in future.